style.css

```diff
@@ -254,3 +256,172 @@ textarea:hover {
 input:hover {
   border: 1px solid #253858;
 }
+
+@media screen and (min-width: 768px) {
+  .main {
+    background: url(images/Header\ bg.svg);
+    background-repeat: no-repeat;
+    background-color: #fff;
+    border-bottom-left-radius: 100px;
+  }
+
+  body {
+    background-color: #e5e5e5;
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-between;
+    padding: 40px 190px 40px 90px;
+    background-color: #fff;
+  }
+
+  .nav-links {
+    display: flex;
+  }
+
+  .nav-links ul {
+    display: flex;
+    list-style: none;
+    gap: 35px;
+  }
+
+  .nav-links ul li a {
+    color: #344563;
+    text-decoration: none;
+    font-size: 15px;
+  }
+
+  .logo {
+    font-size: 18px;
+  }
+
+  h1 {
+    font-size: 45px;
+  }
+
+  .fa-bars {
+    visibility: hidden;
+  }
+
+  .presentation {
+    font-size: 20px;
+    padding-left: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .lang {
+    padding-top: 0;
+    padding-bottom: 0;
+  }
+
+  .connect {
+    font-size: 16px;
+    padding-left: 0;
+  }
+
+  .icons ul li a {
+    font-size: 20px;
+  }
+
+  .icons {
+    margin-left: 0;
+    padding: 12px;
+  }
+
+  .content {
+    width: 85%;
+  }
+
+  .works {
+    background-color: #fff;
+    display: flex;
+    width: 70%;
+    margin-bottom: 100px;
+    margin-top: 100px;
+    height: 410px;
+  }
+
+  .part {
+    height: 350px;
+  }
+
+  #section2,
+  #section4 {
+    flex-direction: row-reverse;
+  }
+
+  .body {
+    margin-left: 250px;
+    width: 100%;
+  }
+
+  .photo {
+    width: 380px;
+    height: 370px;
+  }
+
+  .contentAndLangs {
+    background-color: #fff;
+    border-top-right-radius: 90px;
+    display: flex;
+    height: 900px;
+  }
```

Comment:
- [ ] As requested by the previous code reviewer: 👇🏽 

> Kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Remember that you have limited chances to approve the project and complete all the requirements to continue forward with the next milestone.**


| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/236584046-2a10a5eb-2139-498d-84a0-e2ad32546841.png) | ![image](https://user-images.githubusercontent.com/58642949/236584079-a067376d-54b3-4367-b33b-775e41ebf4f4.png) |